cpp/include/cudf/io/csv.hpp

```diff
@@ -374,7 +374,7 @@ class csv_reader_options {
   {
     if ((size != 0) and ((_skiprows != 0) or (_skipfooter != 0) or (_nrows != -1))) {
       CUDF_FAIL(
-        "When ther is valid value in skiprows or skipfooter or nrows, range size can't have "
+        "When there is valid value in skiprows or skipfooter or nrows, range size can't have "
         "non-zero value");
```

Comment:
While we are at it...

```suggestion
        "If the value of any of skiprows, skipfooter or nrows is valid, range size cannot be non-zero.");
```